README.md

```diff
@@ -1,78 +1,65 @@
+* **Database password**, `MYSQL_PASSWORD=<db-password>`: The password for database user.
+* **Database root password** `MYSQL_ROOT_PASSWORD=<db-root-password>`: The root password for the MariaDB instance.
+* **Database name** `MYSQL_DATABASE=<db-name>`: The name of the database.
+* **Application name**, `MATOMO_APP_NAME=<app-name>`:  Unique identifier for your application. 
+  It is advisable to use your username as the recommended value.
+* **Username**, `MATOMO_USERNAME=<app-username>`: Create a new username for logging into Matomo
+* **Password**, `MATOMO_PASSWORD=<app-password>`: Create a new password for logging into Matomo
+* **Matomo database username**, `MATOMO_DATABASE_USER=<db-username>`: Same sername that was
+  specified for MariaDB.
+* **Matomo database password**, `MATOMO_DATABASE_PASSWORD=<db-password>`: Same password
+  (non-root) that was specified for MariaDB.
+* **Matomo database name**, `MATOMO_DATABASE_NAME=<db-name>`: Same database name that was
+  specified MariaDB.
 
-### Variables
-
-* **Application name**, `NAME=<app-name>`: Unique identifier for your application,
-  recommended value - your username
-* **Username**, `USERNAME=<app-username>`: Create a new username for logging into Matomo
-* **Password**, `PASSWORD=<app-password>`: Create a new password for logging into Matomo
-* **Application hostname suffix**, `APPLICATION_DOMAIN_SUFFIX=<app-suffix>`: The exposed
-  hostname suffix that will be used to create routes for the application. Default when
-  using Rahti is `rahtiapp.fi`
-* **Matomo database username**, `MATOMO_DATABASE_USER=<db-username>`: Username that was
-  specified when deploying MariaDB in the previous step
-* **Matomo database password**, `MATOMO_DATABASE_PASSWORD=<db-password>`: Password
-  (non-root) that was specified when deploying MariaDB in the previous step
-* **Matomo database name**, `MATOMO_DATABASE_NAME=<db-name>`: Database name that was
-  specified when deploying MariaDB in the previous step
 
 ### Command-line deployment of Matomo
 
 1. Download the `matomo-template.yaml` from from this repo.
 2. Process and apply template using default values from template and passing you application
-   specific parameters.
+   specific parameters, sample example shown below:-
 
 ```bash
-oc process -f matomo-template.yaml -p NAME=<app-name>  -p USER=<app-username> PASSWORD=<app-password>  -p MATOMO_DATABASE_USER=<db-username> -p MATOMO_DATABASE_PASSWORD=<db-password> -p MATOMO_DATABASE_NAME=<db-name> | oc apply -f -
+oc process -f matomo-template.yaml -p MATOMO_APP_NAME="matomo-app-johndoe" -p MYSQL_USER="matomouser" -p MYSQL_PASSWORD="matomo" -p MYSQL_ROOT_PASSWORD="letmeinroot" -p MYSQL_DATABASE="matomodb" -p MATOMO_USERNAME="matomo" -p MATOMO_PASSWORD="matomopass" -p MATOMO_DATABASE_USER="matomouser"  -p  MATOMO_DATABASE_PASSWORD="matomo" -p  MATOMO_DATABASE_NAME="matomodb" | oc apply -f 
 ```
```

Comment:
It misses the dash `-` at the end of the command `oc apply -f`.  
It should be `oc apply -f -`